start.go

```diff
@@ -85,12 +85,17 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 	},
 }
 
+func init() {
+	if len(os.Args) > 1 && os.Args[1] == "init" {
+		runtime.GOMAXPROCS(1)
```

Comment:
I don't think it's necessary, since once we lock to an OS thread, we're assured that `pid == task_get_vnr(current)` -- even with Go's funky syscall semantics.